Frontend/ui-customer/src/Resources/Localizable/Strings.purs

```diff
@@ -16,27 +16,27 @@
 module Language.Strings where
 
 import Data.String.Common (trim)
-import Language.Types (STR, getKeyString)
-import MerchantConfig.Utils (getENStrings, getStringFromConfig, getStringWithVar)
+import Language.Types (STR)
+import MerchantConfig.Utils (getStringFromConfig, getStringWithVar)
 import Prelude (($))
+import Data.Maybe (Maybe(..))
+import Debug (spy)
+import ConfigJBridge (getKeyInSharedPrefKeysConfig)
+import Resources.Localizable.BN (getBN)
+import Resources.Localizable.EN (getEN)
+import Resources.Localizable.HI (getHI)
+import Resources.Localizable.KN (getKN)
+import Resources.Localizable.ML (getML)
 
-data LANGUAGE_KEY = EN_US | KN_IN | HI_IN | BN_IN | ML_IN
-
-getKey :: String -> LANGUAGE_KEY
-getKey key = do
-    case key of 
-        "EN_US" -> EN_US
-        "KN_IN" -> KN_IN
-        "HI_IN" -> HI_IN
-        "BN_IN" -> BN_IN
-        "ML_IN" -> ML_IN
-        _ -> EN_US
---TODO: Translate in OR AS
 getString :: STR -> String
-getString key = getStringFromConfig $ trim $ getKeyString key
-
-getVarString :: STR -> Array String -> String
-getVarString key vals = getStringWithVar (trim $ getKeyString key) vals
-
-getEN :: STR -> String
-getEN key = getENStrings $ trim $ getKeyString key
\ No newline at end of file
+getString key = 
+  let language = getKeyInSharedPrefKeysConfig "LANGUAGE_KEY"
```

Comment:
`getString` function is called almost for every string we use in our screens. Assuming the frequency of this function, is it okay to call `SharedPref` everytime? Can we cache the language for each session in our state?

Reply:
yes we can cache per session but depending on state may not be optimal as we have to make sure state is updated all the time

Reply:
we can store the value in `js` for that while updating native shared pref will update `js` also, so while fetching value we will call `js` and not native

Reply:
what about keeping it in GlobalProps? is it not safer to store in GlobalProps?